core/client/src/main/java/org/phenotips/remote/client/internal/RemoteMatchFinder.java

```diff
@@ -78,6 +96,47 @@
     @Inject
     private ConsentManager consentManager;
 
+    @Inject
+    private Provider<XWikiContext> provider;
+
+    private Date completedTime;
+
+    private XWikiDocument prefsDoc;
+
+    private List<String> remoteIdsList;
+
+    @Override
+    public void initialize() throws InitializationException
+    {
+        this.setRemotesList();
```

Comment:
it is a more clear pattern to have `this.remoteIdsList = this.getRemotesList()`, it is best to minimize the number of times a method sets global/class variables